WD-Firmware/examples/ex-freertos/main.c

```diff
@@ -91,7 +95,7 @@ here. */
 #include <unistd.h>
 #include "platform.h"
 #include "encoding.h"
-#include "plic/plic_driver.h"
+#include "plic_driver.h"
 

```

Comment:
add headers (comments for sections)

Reply:
Irrelevant as this file is being abandoned